Sources/TokamakDOM/DOMFiberRenderer.swift

```diff
@@ -15,6 +15,7 @@
 //  Created by Carson Katri on 2/7/22.
 //
 
+import Foundation
 import JavaScriptKit
 @_spi(TokamakCore) import TokamakCore
 @_spi(TokamakStaticHTML) import TokamakStaticHTML
```

Comment:
nit: making formatting consistent for `@_spi` attributes while we're at it 🙂
```suggestion
@_spi(TokamakCore) 
import TokamakCore
@_spi(TokamakStaticHTML) 
import TokamakStaticHTML
```

Reply:
Is it possible to configure this in swiftformat so it does it on commit?

Reply:
There's [`wrapAttributes` rule](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapattributes), but I don't see options for `import` declarations there. Still would help with the rest of our code, we need `prev-line` value set for `--funcattributes`, `--typeattributes`, and `--varattributes`.

Reply:
I could do this in a separate PR so we can reformat all the files without this PR getting too cluttered.

Reply:
No worries, I'll do that now.